site/docs/v1/tech/apis/_user-comment-search-request-parameters.adoc

```diff
@@ -0,0 +1,42 @@
+
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}comment# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the User Comment text. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `comment` field.
+
+[field]#{parameter_prefix}numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#{parameter_prefix}orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `name ASC`#::
+The field to order the search results as well as an order direction.
++
+The possible values are:
++
+ * `comment` - the User Comment text
+ * `id` - the UUID of the user Comment
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the User Comment was created
+ * `tenantId` - the UUID of the Tenant to which the User Comment belongs
```

Comment:
This seems weird too.

Reply:
Weird to involve tenant in sorting? Or weird to sort by Id?

Reply:
I'm trying to imagine a case where I'd want to sort by tenant Id. I can imagine wanting to filter by tenant Id, for sure, but I guess if you are pulling all the comments across the entire instance, you might want to sort them by tenant Id. 

Again, since the code and doc is written, seems fine to leave it in. I just don't get the use case.